src/scenes/Event/components/Button.js

```diff
@@ -6,8 +6,10 @@ export const Button = withStyles(styles, { withTheme: true })(({ classes, ...pro
   <_Button
     fullWidth
     size='large'
+    color='default'
```

Comment:
why default? we normally use `primary` to use the defined primary color from the theme.